src/components/sticky-object.js

```diff
@@ -82,70 +93,3 @@ AFRAME.registerComponent("sticky-object", {
     }
   }
 });
-
-AFRAME.registerComponent("sticky-object-zone", {
```

Comment:
I removed this because I'm a bad person but if you reprimand me I will bring it back.